src/ui/profile_preferences.ui

```diff
@@ -55,6 +55,19 @@
        <string>General</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_19">
```

Comment:
I'm not in support of THAT - (don't the tabs scroll if there are more than can be fit into the space on the widget?) - I think of the "Special options" as the area where dragons hang out and normal users should not be playing in there... 😈

OTOH are the current names still appropriate:
![snapshot8](https://user-images.githubusercontent.com/6163092/27153351-08548f4a-5149-11e7-9da3-4bc40dc0e85b.png)

I think we could make following changes:
"Input line" --> "Input"
"Main Display" --> "Display"
"Color view" --> "Display Colors"
"Mapper" --> "Map"
"Mapper colors" --> "Map Colors"
"Special options" --> "Special"

:warning: __I18n__ :warning: However we must remember that the texts for other languages could be longer that the texts that American English uses...! {E.g. British "Colour" not "Color"}